src/lib.rs

```diff
@@ -374,7 +374,7 @@
 //! Advanced time zone handling is not yet supported.
 //! For now you can try the [Chrono-tz](https://github.com/chronotope/chrono-tz/) crate instead.
 
-#![doc(html_root_url = "https://docs.rs/chrono/latest/")]
+#![doc(html_root_url = "https://docs.rs/chrono/latest/", test(attr(deny(warnings))))]
```

Comment:
In order to allow (at least in theory) each commit to pass CI, this should be added somewhere near the end, not near the beginning.